tensordict/utils.py

```diff
@@ -1862,3 +1862,65 @@ def _index_preserve_data_ptr(index):
     if isinstance(index, slice) and (index.start == 0 or index.start is None):
         return True
     return False
+
+
+def remove_duplicates(
+    tensordict: TensorDictBase, key: str, dim: int = 0
+) -> TensorDictBase:
+    """Removes indices duplicated in 'key' along the specified dimension.
+
+    This method detects duplicate elements in the tensor associated with the specified 'key' along the specified
+    'dim' and removes elements in the same indices in all other tensors within the TensorDict.
```

Comment:
```suggestion
    This method detects duplicate elements in the tensor associated with the specified `key` along the specified
    `dim` and removes elements in the same indices in all other tensors within the TensorDict.
```

Reply:
We should make it clear that this all happens along the batch-size (ie, empty batch size will throw an error etc)